buildenv/jenkins/JenkinsfileBase

```diff
@@ -511,6 +511,8 @@ def setup() {
 		VENDOR_TEST_BRANCHES = (params.VENDOR_TEST_BRANCHES) ? "--vendor_branches \"${params.VENDOR_TEST_BRANCHES}\"" : ""
 		VENDOR_TEST_DIRS = (params.VENDOR_TEST_DIRS) ? "--vendor_dirs \"${params.VENDOR_TEST_DIRS}\"" : ""
 		VENDOR_TEST_SHAS = (params.VENDOR_TEST_SHAS) ? "--vendor_shas \"${params.VENDOR_TEST_SHAS}\"" : ""
+		env.IS_SVT_TESTREPO = params.VENDOR_TEST_REPOS.contains('SVTTestRepo') ? 'true' : 'false'
```

Comment:
Last review I missed this one, all other `env` parameters use `true` instead of `"true"`, is there a reason we need to use quotation marks here?

Reply:
No, there was no reason as such . I have made the change to use `true` instead of `"true"`